webrtc-stats.html

```diff
@@ -2049,6 +2050,28 @@ <h2>
                   MediaStreamTrack, if <code>type</code> is <code>"track"</code>).
                 </p>
               </dd>
+              <dt>
+                <dfn><code>hugeFramesSent</code></dfn> of type <span class="idlMemberType">
+                <a>unsigned long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Represents the total number of huge frames sent by this RTCRtpSender (or for this
+                  MediaStreamTrack, if <code>type</code> is <code>"track"</code>). Huge frames, by
+                  definition, are frames with encoded size at least 2.5 times the typical frame size,
+                  where the typical frame size is defined as the target bitrate per second divided by
+                  the target fps set for the encoder at the time the frame was encoded. These are
+                  usually difficult to encode frames with a lot of changes in the picture. This can be
```

Comment:
complex instead of difficult.

I do not know what difficult means in this case.